src/MinimalDelegation.sol

```diff
@@ -126,7 +133,7 @@ contract MinimalDelegation is IERC7821, IKeyManagement, ERC1271, EIP712, ERC4337
     }
 
     function _authorizeCaller() private view {
```

Comment:
can you add a dev comment saying that this function should only be called if there is an enforcing locker check , since otherwise msgSender() would not be set